lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -2434,11 +2568,17 @@ def __init__(self, **kwargs):
         if self.module.params.get("runtime") and not runtime_supported:
             self.fail('docker API version is %s. Minimum version required is 1.12 to set runtime option.' % (docker_api_version,))
 
+        healthcheck_supported = LooseVersion(docker_version) >= LooseVersion('2.0')
+        if self.module.params.get("healthcheck") and not healthcheck_supported:
+            self.module.warn("docker or docker-py version is %s. Minimum version required is 2.0 to set healthcheck option. "
```

Comment:
I think we should better fail here? It will be more consistent with other non-supported options.

Reply:
Yes, it should be `fail` instead of `warn`.